tests/benchmarks/cpu/find_volume.cpp

```diff
@@ -0,0 +1,80 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "detray/detectors/create_toy_geometry.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/host_memory_resource.hpp>
+
+// Google include(s).
+#include <benchmark/benchmark.h>
+
+// System include(s).
+#include <iostream>
+
+// Benchmarks the cost of searching a volume by position
+void BM_FIND_VOLUMES(benchmark::State &state) {
+
+    // This test is broken at the moment, don't run it.
+    state.SkipWithError("Benchmark disabled for the toy geometry");
+    return;
```

Comment:
This is because of #256. As discussed on Mattermost, this benchmark is crashing for the moment. :frowning: